README.md

```diff
@@ -165,10 +165,6 @@ If you have a problem or suggestion, please [open an issue](https://github.com/g
 
 Emissary is licensed under the [MIT license](LICENSE).
 
-The MIT license grant is not for GitHub's trademarks, which include the logo designs. GitHub reserves all trademark and copyright rights in and to all GitHub trademarks. GitHub's logos include, for instance, the stylized designs that include "logo" in the file title in the following folder: https://github.com/github/emissary/tree/master/docs/logos/
-
-GitHub® and its stylized versions and the Invertocat mark are GitHub's Trademarks or registered Trademarks. When using GitHub's logos, be sure to follow the GitHub logo guidelines.
-
```

Comment:
I could not find the URL for the logo, so I removed the description related to the logo.